### PR TITLE
Battery temperature display 176

### LINKED ISSUE #176: Fix battery temperature display on PDS page
The html/css of the battery temperature display are badly written and were hidden because of invalid feedback from the PDS. This should be corrected in the future. The invalid PDS feedback should be corrected in #126 but the html/css should be corrected here.

"It says `T1` for `battery-temp-2` and `T2` for `battery-temp-1` but `T3` for `battery-temp-3`
Kind of confusing"

_Originally posted by @MartensCedric in https://github.com/_render_node/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkMTk3Njc0MTI5OnYy/pull_request_review_threads/discussion_

"Also I think a nice to have would be to simply add a hover to explain what the battery temperature is for. (Like what does T1 correspond to, T2 and T3)

This can be done with `<div title="Text that is hovering"></div>`"

_Originally posted by @MartensCedric in https://github.com/space-concordia-robotics/robotics-prototype/pull/164_

### PR DESCRIPTION
**Pull request by badgerwild**
Closes #176 

- Fixes inaccurate battery labels and adds hover text to battery label. More meaningful text can be added later, as required.
- Also, temporary fix to the T1 / Battery overlap issue on GUI.

### CODE REVIEW

**Comment by NichHarris** [comment]
> Did you talk to the elec team to know what is the meaning of these mysterious temperatures?

Yeah I talked to Jeff about them. As I had assumed, for right now, none of them have a specific use other than just temperature measurement. I suggested to just add the hover even though it would be repetitive I guess. If the temp sensors ever get specific purpose the hover can be updated.

File: robot/basestation/static/css/bootstrap-4.1.2.min.css [hunk 1]
```diff
@@ -3,5 +3,5 @@
...
...s-track{width:100%;height:.5rem;color:transparent;cursor:pointer;background-color:transparent;border-color:transparent;border-width:.5rem}.custom-range::-ms-fill-lower{background-color:#dee2e6;border-radius:1rem}.custom-range::-ms-fill-upper{margin-right:15px;background-color:#dee2e6;border-radius:1rem}.custom-control-label::before,.custom-file-label,.custom-select{transition:background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out}@media screen and (prefers-reduced-motion:reduce){.custom-control-label::before,.custom-file-label,.custom-select{transition:none}}.nav{display:-ms-flexbox;display:flex;-ms-flex-wrap:wrap;flex-wrap:wrap;padding-left:0;margin-bottom:0;list-style:none}.nav-link{display:block;padding:.5rem 1rem}.nav-link:focus,.nav-link:hover{text-decoration:none}.nav-link.disabled{color:#6c757d}.nav-tabs{border-bottom:1px solid #dee2e6}.nav-tabs .nav-item{margin-bottom:-1px}.nav-tabs .nav-link{border:1px solid transparent;border-top-left-radius:.25rem;border-top-right-radius:.25rem}.nav-tabs .nav-link:focus,.nav-tabs .nav-link:hover{border-color:#e9ecef #e9ecef #dee2e6}.nav-tabs .nav-link.disabled{color:#6c757d;background-color:transparen
```
Comment (PeterGhimself) [inline robot/basestation/static/css/bootstrap-4.1.2.min.css]: This change looks accidental, it should be reverted

Comment (badgerwild) [reply robot/basestation/static/css/bootstrap-4.1.2.min.css]: reverted to previous version

Comment (PeterGhimself) [reply robot/basestation/static/css/bootstrap-4.1.2.min.css]: Thanks

**Comment by PeterGhimself** [comment]
The revert you did seems fine, but the merge conflict still stands in `robot/basestation/templates/Navbar.html`.

**Comment by badgerwild** [comment]
> conflict

tired to fix merge conflict by pulling from origin.

**Comment by MartensCedric** [comment]
Please resolve conflicts

**Comment by MartensCedric** [comment]
> > conflict
> 
> tired to fix merge conflict by pulling from origin.

Perhaps it didn't work. Do you need help to merge this?

**Comment by badgerwild** [comment]
> > > conflict
> > 
> > 
> > tired to fix merge conflict by pulling from origin.
> 
> Perhaps it didn't work. Do you need help to merge this?

I think I do.

**Comment by badgerwild** [comment]
> As brought up by @vashmata in person, stated in the issue #176 "the html/css should be corrected here".
> The CSS still needs to be fixed.

Fixed this in latest commit.

**Pull request merged**